server/src-lib/Hasura/RQL/DDL/Relationship.hs

```diff
@@ -151,6 +154,51 @@ createObjRelP2 (WithTable qt rd) = do
   objRelP2 qt rd
   return successMsg
 
+runCreateRemoteRelationship ::
+     (MonadTx f, CacheRM f) => CreateRemoteRelationship -> f EncJSON
+runCreateRemoteRelationship createRemoteRelationship = do
+  runCreateRemoteRelationshipP1 createRemoteRelationship
+  runCreateRemoteRelationshipP2 createRemoteRelationship
+
+runCreateRemoteRelationshipP1 ::
+     (MonadTx f, CacheRM f) => CreateRemoteRelationship -> f ()
```

Comment:
This functions uses `askSchemaCache` which has a monad class contraint. Should we change `f` -> `m` ?

Reply:
:man_shrugging: Sure.